js/private/js_run_devserver.mjs

```diff
@@ -1,27 +1,163 @@
-import * as path from 'node:path'
-import * as perf_hooks from 'node:perf_hooks'
-import * as fs from 'node:fs'
-import * as os from 'node:os'
-import * as child_process from 'node:child_process'
-import * as crypto from 'node:crypto'
-import * as readline from 'node:readline'
-
+import * as path from 'node:path';
```

Comment:
Maybe a note that this is now a generated file

Reply:
The `write_source_file(s)` rules directly copy from the tsc/rollup-generated code to src so we can't edit it, at least not without some extra `genrule` logic which I don't think is worth it atm. Tests will fail if you edit this file is not a direct clone of the generated file so I'm not too worried about it being bad or out of date...